spring-cloud-gcp-examples/spring-cloud-gcp-sql-example/src/main/resources/data.sql

```diff
@@ -0,0 +1,4 @@
+INSERT INTO users VALUES
+  ("luisao@gmail.com", "Anderson", "Silva"),
```

Comment:
These names and emails seem t0o realistic IMO.
Maybe at least change emails from `@gmail.com` to `@example.com`.